### PR TITLE
feat: added typography

### PR DESCRIPTION
**Pull request by TejasShekar**
### Things to be noted : 
- Site not responsive yet.
- Code blocks have not been added yet.
- Review only the files from the Typography folder. All other files can be ignored as they are simple path updates for typography component and minor code refactor.

### CODE REVIEW

**Review by kotesh-arya — APPROVED** [review]
Everything looks good for a merge 👍️

File: Components/Typography/index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      </main>
+    </div>
+    <footer class="footer mb-1">
+      Made by
+      <a href="https://github.com/TejasShekar" target="_blank">
+        <span class="span">Tejas</span>
+      </a>
+    </footer>
+    <script src="../../Stylesheets/theme-toggle.js"></script>
+    <script src="./modal.js"></script>
```
Comment (xii-vi) [inline Components/Typography/index.html]: You don't need a modal in typography.

**Review by xii-vi — APPROVED** [review]
_Go through comments before merging.
rest is okay to merge._

File: Components/Typography/index.html (hunk 1 above)
Comment (TejasShekar) [reply Components/Typography/index.html]: Thanks for pointing out. Will remove it.

**Pull request merged**